diff -ur -N OpenOffice.org1.1_SDK.orig/settings/settings.mk OpenOffice.org1.1_SDK/settings/settings.mk
--- OpenOffice.org1.1_SDK.orig/settings/settings.mk	Mon Jul 28 18:18:34 2003
+++ OpenOffice.org1.1_SDK/settings/settings.mk	Sat Jan 17 21:49:20 2004
@@ -14,6 +14,12 @@
 ifeq "$(PLATFORM)" "WindowsNT"
 PLATFORM = windows
 endif
+ifeq "$(PLATFORM)" "MINGW32_NT-5.0"
+PLATFORM = windows
+endif
+ifeq "$(PLATFORM)" "MINGW32_NT-5.1"
+PLATFORM = windows
+endif
 ifeq "$(PLATFORM)" "CYGWIN_NT-5.0"
 PLATFORM = windows
 endif
